File: Maths_And_Stats/Statistics/Median/Median.js
var data = [7, 9, 12, 13, 16];
var length = data.length;
var position = 0;
var median = 0;
data.sort(function (a, b) {
  return a - b;
});
if (length % 2 === 0) {
  position = Math.ceil(length / 2) - 1;
  median = (data[position] + data[position + 1]) / 2;
} else {
  position = Math.floor(length / 2);
  median = data[position];
}
console.log("Median is : ", median);

File: Maths_And_Stats/Statistics/Median/Median.py
# Python program for finding median 

# list of elements to calculate median 
data = [7, 2, 3, 6, 7] 

# length of the list
length = len(data) 

# Sort list
data.sort() 

# check if the length of list is odd or even
if length % 2 == 0: 
	median1 = data[length//2] 
	median2 = data[length//2 - 1] 
	median = (median1 + median2)/2
else: 
	median = data[length//2] 
	
# Print Median
print("Median is: " + str(median)) 